lib/transcript.yml

```
@@ -34,19 +34,19 @@ messages:
 
       *Next step:* Type \`/scrappy displaystreaks\` to display your streak in your status!
     2: |
-      Right on for day two! I'll add that to your scrapbook at ${this.scrapbookLink}.
+      Day two already! It's felt like 100! Keep it up! I'll add that to your scrapbook at ${this.scrapbookLink}.
 
       Did you know you can add custom CSS to your scrapbook? Join <#C015M6U6JKU> to learn how!
     3: |
-      I'll scarf that down and add it to your scrapbook for day 3. You're getting up there! ${this.scrapbookLink}
+      I'll scarf that down and add it to your scrapbook for day 3. You're moving up the ranks! ${this.scrapbookLink}
 
       Did you know you can add a custom domain to your scrapbook? Check out <@U0266FRGP>\'s: https://zachlatta.com
       Add your own domain by typing \`/scrappy setdomain\`.
-    4: Ding dong! That's day 4!!! ${this.scrapbookLink}
-    5: 5 whole days of streaks! You're doing amazing sweaty!!! ${this.scrapbookLink}
-    6: 6 days! You've almost made it to a week!!! Don't stop now!!!! ${this.scrapbookLink}
+    4: Hello, hello? Uh, I wanted to let you know that that's day 4 completed!!! ${this.scrapbookLink}
+    5: 5 whole days of streaks! That's like one more than yesterday!!! ${this.scrapbookLink}
+    6: 6 days! You've almost made it to a week!!! Don't stop now!!!! A week is 7 days by the way. ${this.scrapbookLink}
     7: |
-      A WHOLE WEEK OF STREAKS?!?! https://www.youtube.com/watch?v=i0dmSIAzWeQ
+      A WHOLE WEEK OF STREAKS?!?! YOU MADMAN!!! https://www.youtube.com/watch?v=i0dmSIAzWeQ
```

```suggestion
      A WHOLE WEEK OF STREAKS?!?! https://www.youtube.com/watch?v=i0dmSIAzWeQ
```
Let’s keep the language gender-neutral :+1: